scan/src/pages/ValidatorIndexPage.re

```diff
@@ -1,702 +1,1269 @@
-          {kvRow(
-             "WEBSITE",
-             {
-               "The validator's website" |> React.string;
-             },
-             {
-               switch (allSub) {
-               | Data((validator, _)) => VExtLink(validator.website)
-               | _ => Loading(100, 16)
-               };
-             },
-           )}
-          <VSpacing size=Spacing.lg />
-          {kvRow(
-             "DETAILS",
-             {
-               "Extra self-added detail about the validator" |> React.string;
-             },
-             {
-               switch (allSub) {
-               | Data((validator, _)) => VDetail(validator.details)
-               | _ => Loading(100, 16)
-               };
-             },
-           )}
-        </div>
-        // <div className=Styles.longLine />
-        // <div className={Styles.fullWidth(`row)}>
-        //   <Col size=1.>
-        //     <Text value="NODE STATUS" size=Text.Lg weight=Text.Semibold />
-        //     <VSpacing size=Spacing.lg />
-        //     {kvRow("UPTIME", VCode(validator.nodeStatus.uptime->Format.fPretty ++ "%"))}
-        //     <VSpacing size=Spacing.lg />
-        //     {kvRow(
-        //        "AVG. RESPONSE TIME",
-        //        VCode(
-        //          validator.avgResponseTime->Format.iPretty
-        //          ++ (validator.avgResponseTime <= 1 ? " block" : " blocks"),
-        //        ),
-        //      )}
-        //   </Col>
-        //   <HSpacing size=Spacing.lg />
-        //   <Col size=1.>
-        //     <Text value="REQUEST RESPONSE" size=Text.Lg weight=Text.Semibold />
-        //     <VSpacing size=Spacing.lg />
-        //     {kvRow("COMPLETED REQUESTS", VCode(validator.completedRequestCount->Format.iPretty))}
-        //     <VSpacing size=Spacing.lg />
-        //     {kvRow("MISSED REQUESTS", VCode(validator.missedRequestCount->Format.iPretty))}
-        //   </Col>
-        // </div>
-        <VSpacing size=Spacing.md />
-        <Tab
-          tabs=[|
-            {
-              name: "PROPOSED BLOCKS",
-              route: Route.ValidatorIndexPage(address, Route.ProposedBlocks),
-            },
-            {name: "DELEGATORS", route: Route.ValidatorIndexPage(address, Route.Delegators)},
-            {name: "REPORTS", route: Route.ValidatorIndexPage(address, Route.Reports)},
-          |]
-          currentRoute={Route.ValidatorIndexPage(address, hashtag)}>
-          {switch (hashtag) {
-           | ProposedBlocks =>
-             switch (validatorSub) {
-             | Data(validator) =>
-               <ProposedBlocksTable consensusAddress={validator.consensusAddress} />
-             | _ => <ProposedBlocksTable.LoadingWithHeader />
-             }
-           | Delegators => <DelegatorsTable address isMobile=false />
-           | Reports => <ReportsTable address />
-           }}
-        </Tab>
-      </div>
-    </Section>;
+  let make = (~status, ~height: ID.Block.t) => {
```

Comment:
I think it's no need to declare the type for height. It should automatically imply from `ID.Block.getRoute`.